egs2/open_li52/asr1/local/data.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+
+# Copyright 2020 Johns Hopkins University (Jiatong Shi)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+# general configuration
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+SECONDS=0
+langs="en de fr cy br cv ky ga-IE sl cnh et mn sah dv sv-SE id ar ta ia lv ja rm-sursilv hsb ro fy-NL el rm-vallader as mt ka or vi pa-IN tt kab ca zh-TW it fa eu es ru tr nl eo zh-CN rw pt zh-HK cs pl uk"
```

Comment:
This is too long.
Also, is it based on some order?
I want to sort the entry in this list with some criteria.

Reply:
Sure, let's do it alphabetically.